stylesheets/_component.timeline.scss

```diff
@@ -0,0 +1,585 @@
+    background: #fafafa;
+    border: 1px solid $timeline-content-border;
+    padding: 1em;
+
+    @include respond-min($screen-tablet) {
+        border: 1px solid $timeline-content-border;
+        margin-left: 70px;
+        margin-top: $padding-base;
+        position: relative;
+
+        &::before,
+        &::after {
+            border: solid transparent;
+            content: '';
+            height: 0;
+            position: absolute;
+            right: 100%;
+            top: 16px;
+            width: 0;
+        }
+
+        &::after {
+            border-color: rgba(136, 183, 213, 0);
+            border-right-color: #fafafa;
+            border-width: 7px;
+            margin-top: -7px;
+        }
+
+        &::before {
+            border-color: rgba(194, 225, 245, 0);
+            border-right-color: $timeline-content-border;
+            border-width: 8px;
+            margin-top: -8px;
+        }
+    }
+
+    &__title {
+        color: #303e49;
+        display: inline-block;
+        margin: 0 0 0 26px;
+
+        @include respond-min($screen-phone) {
+            margin: 0 0 0 32px;
+        }
+
+        @include respond-min($screen-tablet) {
+            line-height: $line-height-base;
+            margin: 0;
+            width: 60%;
+        }
+    }
+
+    &__note-title {
+        margin: 1em 0;
+    }
+
+    &__text {
+        @include word-wrap();
+
+        margin: .5em 0;
+
+        @include respond-min($screen-tablet) {
+            margin: 1em 0;
+        }
+
+        a {
+            word-break: break-all;
+        }
+
+        .avatar {
+            background-color: color(white);
+            vertical-align: bottom;
+        }
+    }
+
+    &__small {
+        color: color(gray, dark);
+        display: inline-block;
+        font-size: .7em;
+
+        .label {
+            font-size: 100%;
+        }
+    }
+
+    &__date {
+        display: block;
+        font-size: 14px;
+        margin-top: $padding-base;
+
+        @include respond-min($screen-tablet) {
+            display: inline-block;
+            float: right;
+            margin-top: 0;
+            text-align: right;
+            width: 40%;
+        }
+    }
+
+    &__attachments {
+        margin: 1em 0;
+
+        li {
+            display: inline-block;
+            margin-right: .5em;
+            width: 150px;
+        }
+    }
+
+    &__thumbnail-link {
+        border-radius: 4px;
+        cursor: pointer;
+        display: inline-block;
+        max-height: 250px;
+        max-width: 100%;
+        overflow: hidden;
+        position: relative;
+
+        @include ie-lte(10) {
+            max-width: 150px;
+        }
+    }
+
+    &__thumbnail {
+        max-height: 250px;
+        max-width: 100%;
+
+        @include ie-lte(10) {
+            max-width: 150px;
+        }
+    }
+
+    &__from {
+        display: inline-block;
+        height: 0;
+        opacity: 0;
+        overflow: hidden;
+        width: 0;
+    }
```

Comment:
if the intention here is to visually hide the element, should we use our existing visually-hidden mixin / utility class for this?